pulp_container/app/access_policy.py

```diff
@@ -20,11 +20,13 @@ def has_namespace_obj_perms(self, request, view, action, permission):
         obj = view.get_object()
         if type(obj) == models.ContainerDistribution:
             namespace = obj.namespace
-            return request.user.has_perm(permission, namespace)
+            return request.user.has_perm(permission) or request.user.has_perm(permission, namespace)
```

Comment:
I think we can call `request.user.has_perm(permission)` just once, right before entering the if-else statement. Then it will look like this:
```
if request.user.has_perm(permission):
    return True

obj = view.get_object()
if type(obj) == models.ContainerDistribution:
    ...
```